source/sdk/structs/HashMap.ooc

```diff
@@ -44,8 +44,8 @@ murmurHash: func <K> (keyTagazok: K) -> SizeT {
 	seed: SizeT = 1
 
 	len := K size
-	m = 0x5bd1e995 : const SizeT
-	r = 24 : const SSizeT
```

Comment:
So it does completely nothing ? Can you even reassign the value after initialization ?